src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let BASE_PRICE = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return BASE_PRICE - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    BASE_PRICE -= SHORT_TERM_DISCOUNT;
```

Comment:
You can i `BASE_PRICE`
Better to return `BASE_PRICE` immediately return 11, as in previous `if`.
If you do this, you can declare `BASE_PRICE` as a `const` instead of `let`